test/utils.js

```diff
@@ -0,0 +1,54 @@
+import expect from 'expect';
+
+import { getCountryName } from '../src/util/countries';
+import { findFilterMatch } from '../src/util/filters';
+
+describe('getCountryName', () => {
+  it("should get name for 'no'", () => {
+    expect(getCountryName('no')).toBe('Norway');
+  });
+
+  it('should return country code if not found', () => {
+    const invalidCountryCode = 'asdadqasda';
+    expect(getCountryName(invalidCountryCode)).toBe(invalidCountryCode);
+  });
```

Comment:
Endre navn på denne? Var litt vanskelig å forstå hva som menes med 'should return country code if not found'